### PR TITLE
Dockerfile updates for SSH container

### PR DESCRIPTION
**Pull request by cblecker**
- Remove unneeded service yaml file
- Clean up and simplify Dockerfile (remove bash configurations)
- Add python3 for sshuttle support

### CODE REVIEW

File: container/Dockerfile [hunk 1]
```diff
@@ -1,26 +1,18 @@
 FROM centos:centos8
 
-# creature comforts
-ENV LANG en_US.UTF-8
-ENV CONTAINER docker 
-ENV USER sre-user
-ENV HOME /home/sre-user
-ENV TERM xterm
```
Comment (yithian) [inline container/Dockerfile:8]: out of curiosity, what's the reason for removing these env variables?

Comment (cblecker) [reply container/Dockerfile:8]: The container isn't interactive, so they aren't needed by any process.

Comment (yithian) [reply container/Dockerfile:8]: ah, ok. i'd assumed they would be picked up by the shell once the user logs in.

**Comment by cblecker** [comment]
/hold cancel

Merging!

**Pull request merged**